src/app/core/components/language-mismatch/language-mismatch.component.html

```diff
@@ -0,0 +1,61 @@
+<p-dialog
+  *ngrxLet="languageMismatch$; let languageMismatch"
+  i18n-header header="Language mismatch"
+  [visible]="!!languageMismatch"
```

Comment:
I may be wrong but I would have made the component  loaded only if `!!languageMismatch`, so using
```
*ngIf="languageMismatch$|async as languageMismatch" 
[visible]="true"
```
Doesn't it work?